docs/resolvers/custom-resolvers.md

```diff
@@ -100,9 +100,54 @@ await context.Note.findBy(args, context);
 
 ### info
 
-The `GraphQLResolveInfo` object is available to every resolver and contains information about the GraphQL schema and the current operation.
+The optional `GraphQLResolveInfo` object is available to every resolver and contains information about the GraphQL schema and the current operation.
 You can optionally pass this to the `GraphbackCRUDService` method. A common use for this parameter is mapping the selected fields from the GraphQL query to the database query. 
 
 ```ts
 await context.Note.findBy(args, context, info);
-```
\ No newline at end of file
+```
+
+:::info
+If you want to map nested the selected fields from the GraphQL query to the database query, you can optionally pass the `nested path` as an argument to the `.findBy(..)` method. 
+
+For example:
+
+If the incoming query is: 
+
+```graphql
+
+query {
+  getDraftNotes {
+    comments {
+      id, 
+      description
+    }
+  }
+}
+```
+
+And somewhere in your custom resolvers, you'd like to map the `id` and `description` fields for the `Comment` model to the underlying database query, you achieve that with:
+
+```ts
+
+// retrieve all notes using incoming args and info
+const { items: allNotes } = await context.Notes.findBy(args, context, info);
+
+const noteIds = allNotes.map(({ id }) => id);
+
+// retrieves comments owned by notes selected above. Specifying the "comments" path so start selection
+const { items: comments } = await context.Comment.findBy({noteId: {in: noteIds}}, context, info, 'comments');
+
+// ... do the rest
+```
+:::
+
+:::caution
```

Comment:
It is strange to see such a large `info` block like this, these are usually reserved for small tidbits of information

![image](https://user-images.githubusercontent.com/11743717/93463900-51629780-f8e0-11ea-9107-c08a8b858c6e.png)

The yellow is hard to read in a large block like that.

Reply:
Good point. Let's make it a sub-section then.